### ISSUE TITLE
Cant install driver

### ISSUE THREAD

**Issue by juanrd0088** [dokan-dev/dokany#935]
Feature request can skip this form. Bug report **must** complete it. `Check List` must be 100% match or it will be automatically closed without further discussion. Please remove this line.

# Environment
 * Windows version: Windows 10
 * Processor architecture: x64
 * Dokany version:  1.4.0.1000 (the last in the changelog)
 * Library type (Dokany/FUSE):  Dokany
 
# Check List
  - [x ] I checked my issue [doesn't exist yet](https://github.com/dokan-dev/dokany/issues?utf8=%E2%9C%93&q=is%3Aissue)
  - [ ] My issue is valid with mirror default sample and not specific to my user-mode driver implementation
  - [ ] I can always reproduce the issue with the provided description below.
  - [ ] I have updated Dokany to the [latest version](https://github.com/dokan-dev/dokany/releases) and have reboot my computer after.
  - [ ] I tested one of the last [snapshot](https://github.com/dokan-dev/dokany/wiki/Build#user-snapshot) from appveyor CI

# Description
After building and installing dokan, i get fails when try to install dokanctl.exe and when i try to run mirror.exe
![image](https://user-images.githubusercontent.com/11312790/96234104-44959a00-0f99-11eb-91ad-091b933f1358.png)

# Logs
Please attach in separate files: mirror [output](https://github.com/dokan-dev/dokany/wiki/How-to-Debug-Dokan#mirrorfs), library logs and kernel [logs](https://github.com/dokan-dev/dokany/wiki/How-to-Debug-Dokan#get-logs).
In case of BSOD, please attach minidump or dump [analyze output](https://github.com/dokan-dev/dokany/wiki/How-to-Debug-Dokan#crash-report-bsod).

**Comment by Liryna** [comment]
Have you reboot after all the bcdedit command?

**Comment by juanrd0088** [comment]
Thanks
I just rebooted and the mirror works but dokanctl fails.
![image](https://user-images.githubusercontent.com/11312790/96235534-347eba00-0f9b-11eb-8fca-94c0c32a90a3.png)


Could dokan1.sys be wrong? Seeing a previous answer I copied it from another path (/ release / driver / sys, instead of release / driver) and it doesn't let me delete it now, because it is active I suppose

**Comment by Liryna** [comment]
Exact, the error is normal at this point.
To replace the driver, you need to `dokanctl.exe /r d`, reboot and then only after you can replace the `dokan1.sys` file and do a `dokanctl.exe /i d` again to start the new version.

Note: That's only needed if you want to make changes in driver. Otherwise you are fine to directly use the one from the installer.
If you do make changes in the driver, I personally use a VM that I restore to the state where the driver is not installer but the bcdedit commands / reboot are already done and then I only need to copy and install the new driver.

**Comment by juanrd0088** [comment]
Thank you very much.
I'm already using a VM but I'll follow your advice.
